website/content/en/preview/getting-started/getting-started-with-karpenter/scripts/step02-create-cluster.sh

```diff
@@ -35,7 +35,7 @@ iamIdentityMappings:
   - system:nodes
 
 managedNodeGroups:
-- instanceType: m5.large
```

Comment:
What's the specific reason that you want this change? I think for most users, the smaller instance type may not necessarily be a good fit due to the resource requirements on Karpenter. We were trying to pick a generic instance type that would support the average case.

Reply:
@jonathan-innis on a two node deployment with a small workload we've not seen karpenter itself use nearly the power required by even a `t3.medium`.

Reply:
I think we are happy with the current default for most customers. The script is mainly meant to be a jumping off point to customize to your use-case. Happy to discuss more on the Kubernetes slack or in the #karpenter-dev channel on the kubernetes slack if you wanted to discuss more

Reply:
sounds like a very bad idea to put a burstable instance in Prod for a client that may not be expecting that to happen.

Reply:
@FernandoMiguel where can I learn more about this concern and why it'd be an issue for karpenter?

Reply:
it's not an issue for karpenter... it's an issue for the workloads
once the CPU credits are exhausted, the CPU baseline can be as low as 20%... this will be a terrible performance